SimpleNtuplizer/plugins/SimpleNtuplizer.h

```diff
@@ -247,6 +260,20 @@ class SimpleNtuplizer : public edm::EDAnalyzer {
   std::vector<Float_t> full5x5_e2x5Bottom_e;
   // H/E
 
+  ////PFCluster
+  Int_t          nClus_;
```

Comment:
Could you follow the same style of the code that Thomas wrote by using _pf in the name of the variable (or any other suffix)?